lib/oidc/options/OAuthOptionsConstructor.ts

```diff
@@ -48,7 +48,7 @@ function assertValidConfig(args) {
       'Required usage: new OktaAuth({issuer: "https://{yourOktaDomain}.com/oauth2/{authServerId}"})');
   }
 
-  if (issuer.indexOf('-admin.') !== -1) {
+  if (issuer.indexOf('-admin.okta') !== -1) {
```

Comment:
AFAICT this will match trex, preview, prod (all TLD's, e.g., com, gov) etc. which is actually a good thing, right?
Could you take a second to consider BYOC testing domains?